cfep-23.md

```diff
@@ -0,0 +1,73 @@
+<table>
+<tr><td> Title </td><td> Migration to the Zulip chat platform </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt;, Matthew R. Becker &lt;becker.mr@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sep 20, 2024</td></tr>
+<tr><td> Updated </td><td> Oct 14, 2024</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/cfep/pull/54 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+The conda-forge core team is proposing to transition chat platforms from Element/Matrix to Zulip.
+
+## Background
+
+Initially, conda-forge offered support through the Gitter platform, which allowed one chat room per repository. `conda-forge/conda-forge.github.io` was chosen for this and many users have received support over the years in the corresponding room. As of September 2024, over 2000 users have joined the room. The conda-forge core team also used private rooms to communicate core issues. However, due to the lack of encryption in Gitter, a Keybase room was created to exchange information about sensitive topics.
+
+When Gitter was integrated in the Matrix protocol and migrated to the Element client, it gained encryption features. This meant that security sensitive items could be discussed in the regular private rooms. As a result, Keybase was abandoned and its contents migrated to a secure secret management vault.
+
+The integration in the Element ecosystem was seen as exciting and positive: emoji reactions were available for the first time, and threads were shortly implemented too. However, over time the excitement faded away, which prompted the proposal discussed in this CFEP.
```

Comment:
This reads a little bit like we got bored and so are leaving 😉 This of course is not what is happening

Would suggest we rewrite this like so to call out there were issues. We need not spell them out here as we already do that below

```suggestion
The integration in the Element ecosystem was seen as exciting and positive: emoji reactions were available for the first time, and threads were shortly implemented too. However, over time usability issues hindered effective communication on Element (described below), which prompted the proposal discussed in this CFEP.
```